test/e2e/airgap.go

```diff
@@ -39,6 +39,21 @@ func runAirgapConfigFlow(test *framework.ClusterE2ETest, localCIDRs string) {
 	test.DeleteCluster()
 }
 
+// runAirgapConfigProxyFlow run airgap deployment but allow bootstrap cluster to access local peers.
```

Comment:
```suggestion
// runAirgapConfigProxyFlow runs air-gap deployment workflow with allowing bootstrap cluster to access local peers.
```

Reply:
lets update the function comment for both  `runAirgapConfigProxyFlow` and `runAirgapConfigFlow`, clarifying what the flow does in details for each case.